docs/how-to/distributed-training.txt

```diff
@@ -83,30 +83,49 @@ S3. In this example, the S3 bucket name is configured via a field
           s3.download_file(s3_bucket, data_file, filepath)
       return download_directory
 
+.. _dtrain-scheduling:
 
 Scheduling Behavior
 -------------------
 
-To use distributed training, ``slots_per_trial`` must be set to a multiple
-of the number of GPUs per machine. For example, if you have a cluster of machines with
-8 GPUs each, you should set ``slots_per_trial`` to a multiple of 8, such as 8, 16, or 24.
-This ensures that the full network and interconnect bandwidth are available to
-multi-GPU workloads and results in better performance.
+The Determined master takes care of scheduling distributed training jobs
+automatically, ensuring that all of the compute resources required for a job are
+available before the job itself is launched. Users should be aware of the
```

Comment:
not exactly true for the k8s RP but probably not worth making this even more difficult for users to understand.